vault/request_handling.go

```diff
@@ -26,6 +26,13 @@ const (
 	replTimeout = 10 * time.Second
 )
 
+// HanlderProperties is used to seed configuration into a vaulthttp.Handler.
+// It's in this package to avoid a circular dependency
+type HandlerProperties struct {
```

Comment:
[Nitpick] Can we call this `HandlerOptions` instead, unless there was a reason to call this `HandlerProperties`?

Reply:
I called it Properties since it's not just user-set options (e.g. Core).